velox/core/QueryConfig.h

```diff
@@ -357,6 +357,14 @@ class QueryConfig {
   /// Empty string if only want to trace the query metadata.
   static constexpr const char* kQueryTraceNodeIds = "query_trace_node_ids";
 
+  /// The max trace bytes limit, if it is zero, then tracing is disabled.
```

Comment:
These changes are nice. Can they be extracted into a separate PR? Can we include these in the PR description?

Reply:
>Can we include these in the PR description?

I've updated these in the PR description.